index.html

```diff
@@ -333,41 +346,39 @@
 								}
 							}
 							activeDistrict = coords[4];
-							if (coords[4] !== '0') {
-								for (i = 500; i <= 5500; i += 1000) {
-									setTimeout(function(){
-										updateStatsBox(filterField);
-									}, i);
-								}
-							} else {
-								document.getElementById('statsBox').style.opacity = 0;
+							for (i = 500; i <= 5500; i += 1000) {
+								setTimeout(function(){
+									updateStatsBox(filterField);
+								}, i);
 							}
 						}, 1500);
 					}
 				}
 			}
 
 			function updateStatsBox(districtType) {
-				if (activeDistrict !== '0') { // only do anything if we have a selected district
-					document.getElementById('statsBox').style.opacity = 1;
-					if (districtType.indexOf("house") > -1) {
-						document.getElementById("stats.districtType").innerText = "House";
-					} else if (districtType.indexOf("senate") > -1) {
-						document.getElementById("stats.districtType").innerText = "Senate";
-					} else {
-						document.getElementById("stats.districtType").innerText = "";
-					}
-					document.getElementById("stats.districtName").innerText = activeDistrict;
-					for (i in loadedPointLayers) {
-						pointsInDistrict = getUniqueFeatures(
-							map.queryRenderedFeatures( { layers:[loadedPointLayers[i][0]] } ),
-							"unique_id"
-						);
-						counterID = "count." + loadedPointLayers[i][0];
-						document.getElementById(counterID).innerText = pointsInDistrict.length;
-					}
+				document.getElementById('statsBox').style.opacity = 1;
+				if (activeDistrict === '0') {
+					document.getElementById("stats.districtType").innerText = "the State of Texas";
```

Comment:
I find my own wording here to be a little clunky, but it was the best I could come up with while in coding mode.  If you want something else you just have to change this string.